omnibus/resources/agent/msi/localbuild/parameters.wxi

```diff
@@ -9,5 +9,6 @@
   <?define EtcFiles="C:/omnibus-ruby/src\etc\datadog-agent" ?>
   <?define ResourcesDir="c:\dev\go\src\github.com\datadog\datadog-agent\omnibus\resources\agent\msi"?>
   <?define IncludePython2= "true" ?>
-  <?define IncludePython3= "false" ?>
+  <?define IncludePython3= "true" ?>
```

Comment:
Is this used for anything? How was python3 working before?

Reply:
this is only used for local builds if you just want to rebuild the installer w/o going through an entire omnibus build (speeds up the code/build/test cycle when working on the install).  It's not used for anything that gets released.